### PR TITLE
Fix failed not working when manually setting as failed in displayShow

### PR DESCRIPTION
**Pull request by fernandog**
@medariox please confirm it fixed. Mine worked as my gist in chat

### CODE REVIEW

**Comment by fernandog** [comment]
@ratoaq2 can you review the queue changes please?

File: medusa/search/queue.py [hunk 1]
```diff
@@ -50,6 +50,14 @@ def is_in_queue(self, show, segment):
         for cur_item in self.queue:
             if isinstance(cur_item, BacklogQueueItem) and cur_item.show == show and cur_item.segment == segment:
                 return True
+            elif isinstance(cur_item, FailedQueueItem) and cur_item.show == show and cur_item.segment == segment:
```
Comment (ratoaq2) [inline medusa/search/queue.py]: A lot of duplicated code 😯

Comment (fernandog) [inline medusa/search/queue.py]: @ratoaq2 yes but how to improve it?

Comment (medariox) [inline medusa/search/queue.py]:
```
isinstance(cur_item, (BacklogQueueItem, FailedQueueItem, etc))
```

**Pull request merged**